app/views/schedules/events.html.haml

```diff
@@ -5,6 +5,13 @@
   %h1.text-center
```

Comment:
The above line `  -if @events_schedules.any?` probably should be removed if this is merged in. It leads to a weird looking case when you have no events in your favorites schedule and the tabs disappear.